xml/art_sles_amd-sev.xml

```diff
@@ -0,0 +1,431 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE appendix
+[
```

Comment:
This should not be an appendix but rather a sect1 (the following sect1 have to be converted to sect2 and so on)

Reply:
Done.